src/main/java/uk/gov/hmcts/reform/sscs/service/SendNotificationService.java

```diff
@@ -288,9 +286,9 @@ private boolean sendBundledAndDocmosisLetterNotification(NotificationWrapper wra
         byte[] associatedCasePdf = null;
         String documentUrl = getBundledLetterDocumentUrl(notificationEventType, newSscsCaseData);
 
+        log.info("documentUrl: " + documentUrl);
```

Comment:
This log statement... Can we add the case id and a better message? Or just remove it?